iot-hub/Samples/service/RoleBasedAuthenticationSample/RoleBasedAuthenticationSample.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.Azure.Devices;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RoleBasedAuthenticationSample
+{
+    /// <summary>
+    /// This sample connects to the IoT hub using Azure active directory credentials and sends a cloud-to-device message.
+    /// </summary>
+    public class RoleBasedAuthenticationSample
+    {
+        public async Task RunSampleAsync(ServiceClient client, string deviceId)
+        {
+            Console.WriteLine("Connecting using token credential.");
+            await client.OpenAsync();
+            Console.WriteLine("Success.");
+
+            Console.WriteLine("Sending a cloud-to-device message.");
+            var str = $"Hello, Cloud!";
+            var message = new Message(Encoding.ASCII.GetBytes(str));
+            await client.SendAsync(deviceId, message);
+            message.Dispose();
```

Comment:
Could you use a using statement here? Like:

```csharp
using var message = new Message(Encoding.ASCII.GetBytes(str));
await client.SendAsync(deviceId, message);
```

Reply:
It would spare you the message.Dispose() line

Reply:
yes thanks :) copy paste somehow blinds me at times :P